Client/Application/LeanplumIntegration.swift

```diff
@@ -108,6 +110,10 @@ class LeanplumIntegration {
         }
         Leanplum.syncResourcesAsync(true)
 
+        if let _ = profile?.prefs.boolForKey(PrefsKeys.HasFocusInstalled) {} else {
```

Comment:
Also, do you really need that check? Don't you simply want to update the `PrefsKeys.HasFocusInstalled` every time the app starts? So that it is always current?